Assets/Scripts/UI/LogText/JumpUpRoll.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Fumbbl.Model;
+using Fumbbl.Model.RollModifier;
+using Fumbbl.Model.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Fumbbl.UI.LogText
+{
+    public class JumpUpRoll : LogTextGenerator<Ffb.Dto.Reports.JumpUpRoll>
+    {
+        public override IEnumerable<LogRecord> Convert(Ffb.Dto.Reports.JumpUpRoll report)
+        {
+            Player player = FFB.Instance.Model.GetPlayer(report.playerId);
+            string neededRoll = "";
```

Comment:
For future reference: `string neededRoll = string.Empty;` is cleaner than a static empty string.